Cargo.toml

```
@@ -21,24 +21,21 @@ is-it-maintained-open-issues = { repository = "mozilla/mtu", branch = "main" }
 maintenance = { status = "actively-developed", branch = "main" }
 
 [dependencies]
-# Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
-libc = { version = "0.2", default-features = false }
-static_assertions = { version = "1.1", default-features = false }
+libc = { version = "0", default-features = false }
```

I don't think `0` would include `0.2`. Given that `libc` is below `1.0`, the move from e.g. `0.1.0` to `0.2.0` is a breaking change. I suggest keeping `0.2` here, allowing consumers of `mtu` to use `mtu` together with `libc` patch releases, e.g. `libc` `0.2.1`.

```suggestion
libc = { version = "0.2", default-features = false }
```